src/snapred/backend/dao/state/DetectorState.py

```diff
@@ -7,6 +7,7 @@ class DetectorState(BaseModel):
     arc: Tuple[float, float]
     wav: float
     freq: float
+    # Expected to only be 0 or 1, potentially should be a bool
     guideStat: int
```

Comment:
``` suggestion
    guideStat: Literal[0, 1]
```

Reply:
This way it's limited to the allowed values, but maybe if we want more possibilities we can easily expand it

Reply:
Did you one better, added an enum.  Through happenstance found out the valid values are apparently 1-2, the story is now updated.